common/static/js/lib/pretty-print.js

```diff
@@ -0,0 +1,97 @@
+/**
```

Comment:
@andy-armstrong yeah I have the same concerns. The library that we are using here is basically for node js. I haven't seen any reputed library with robust testing apart from [js-beautify](https://github.com/beautify-web/js-beautify). But I think that would be an overkill for just formatting the xml. it's humungous near 100 KB library.
Apart from this we have libraries like the one we are using with no tests.

I have a test to test formatting of xml in capa problem but I can add more tests specific to the library.